introduction.md

```diff
@@ -0,0 +1,4 @@
+Introduction
```

Comment:
Added comment to add Ruby as my favorite Code School class as well as Mastering GitHub.